elements/bulbs-video/components/revealed.js

```diff
@@ -97,6 +97,10 @@ export default class Revealed extends React.Component {
       videoMeta.player_options.defaultCaptions = true;
     }
 
+    if (this.props.embedded) {
+      videoMeta.player_options.embedded = true;
+    }
+
```

Comment:
you could do the coercion trick if you're paranoid `videoMeta.player_options.embedded = !!this.props.embedded`